modules/cms/controllers/themes/_theme_list_item.htm

```diff
@@ -8,7 +8,7 @@
 <div class="layout-cell min-height theme-description">
     <h3><?= e($theme->getConfigValue('name', $theme->getDirName())) ?></h3>
     <?php if (strlen($author)): ?>
-        <p class="author"><?= str_replace(':name', '<a href="'.e($theme->getConfigValue('homepage', '#')).'">'.e($author).'</a>', e(trans('cms::lang.theme.by_author'))) ?></p>
+        <p class="author"><?= trans('cms::lang.theme.by_author', ['name' => '<a href="'.e($theme->getConfigValue('homepage', '#')).'">'.e($author).'</a>']) ?></p>
```

Comment:
Don't escape everything individually, do it all at once.
Example:
```php
<?= e(trans('cms::lang.theme.by_author', ['name' => "<a href='" . $theme->getConfigValue('homepage', '#')) . "'>$author</a>"]); ?>
```
Make sure you apply that style to everything in this PR. Don't individually escape data, just do it once.